docs-ref-conceptual/azure-cli-configuration.md

```diff
@@ -98,7 +98,7 @@ When you provide a default value, that argument is no longer required by any com
 
 | Section | Name      | Type | Description|
 |---------|-----------|------|------------|
-| __core__ | output | string | The default output format. Can be one of `json`, `jsonc`, `tsv`, or `table`. |
+| __core__ | output | string | The default output format. Allowed values: `json` (default), `jsonc`, `yaml`, `yamlc`, `table`, `tsv`, `none`. |
```

Comment:
I resorted the allowed values to align with https://learn.microsoft.com/en-us/cli/azure/format-output-azure-cli. I think we can also add:

```suggestion
| __core__ | output | string | The default output format. Allowed values: `json` (default), `jsonc`, `yaml`, `yamlc`, `table`, `tsv`, `none`. For more information, see [Output formats for Azure CLI commands](format-output-azure-cli.md). |
```